pages/database-management/authentication-and-authorization/auth-system-integrations.mdx

```diff
@@ -9,42 +9,60 @@ import { Tabs, Tab } from 'nextra/components'
-the returned verdict, Memgraph will either close the connection to the
-connected user or will allow the connection and associate the connection 
-with the appropriate role.
-
-When Memgraph is switched to use the external auth module for authentication
-its internal users are automatically disabled. All users are authenticated only
-using the module, existing local users are ignored. In order to authorize query 
-execution, the auth module must return a role associated with the user. This 
-role must be registered locally on Memgraph.
+Memgraph supports authentication and authorization using external auth modules.
+It includes built-in support for basic (username and password) authentication
+via LDAP, as well as single sign-on over the SAML and OIDC protocols.
+
+When a user is connecting, Memgraph will forward the user’s supplied
+credentials (username and password, or scheme and auth response) to the
+external auth module and retrieve its verdict. Based on this verdict, Memgraph
+will either allow the user to connect or, in case of failure, close the
+connection.
 
 As this is an Enterprise feature, once the Memgraph Enterprise license expires,
 newly created users will be granted all privileges. The existing users'
 privileges will still apply but you won't be able to manage them.
 
-## Configuration flags
+## Configuration
+
+### Roles
+
+User roles must be defined in Memgraph before using auth modules because these
+modules return the role associated with the user.
+
+### Flags
 
 Use the following configuration flags to configure the external auth module
 authentication and authorization mechanisms used by Memgraph.
 
  Flag                               | Description
 ------------------------------------|------------
- `--auth-module-executable`         | Path to the executable that should be used for user authentication/authorization.
+ `--auth-module-mappings`           | Associates auth schemes to external modules. A mapping is structured as follows: `<scheme>:<absolute path to module>` and individual entries are separated with `;`. If the mapping contains whitespace, enclose the flag value with quotation marks.
+ `--auth-module-executable`         | [DEPRECATED] Path to the executable that should be used for user authentication/authorization. Replaced by `--auth-module-mappings=basic` from Memgraph 2.18.
  `--auth-module-timeout-ms`         | Specifies the maximum time that Memgraph will wait for a response from the external auth module.
  `--auth-password-permit-null`      | Can be set to false to disable null passwords.
  `--auth-password-strength-regex`   | The regular expression that should be used to match the entire entered password to ensure its strength.
 
-## Communication
+Auth schemes supported by Memgraph out-of-the-box have default module paths. To
+enable them, you don’t need to specify the auth module path, e.g.
+`--auth-module-mappings=basic;saml-entra-id;oidc-okta`. This applies to the
+following schemes:
```

Comment:
This is hard to understand. The example is given in the negative sense ("you don't need to...") which is confusing to the reader. Tell the reader what to do to be successful.

Reply:
Thanks, applied